app/partials/wizardNav.html

```diff
@@ -0,0 +1,9 @@
+<nav class="nav-wizard">
+    <ol role="tablist">
+        <li ng-repeat="step in steps" ng-class="step.status" role="tab" tabindex="0">
+            <span class="icon">{{$index+1}}</span>
```

Comment:
This is minor, but couldn't you be more explicit with the icon span by defining `step.icon` in the wizard directive instead of relying on a scoped $index and adding 1?